configuration-files/aws-provided/instance-configuration/storage-efs-mountfilesystem.config

```diff
@@ -79,7 +79,6 @@ files:
                 echo 'ERROR: Directory creation failed!'
                 exit 1
             fi
-            chmod 777 ${EFS_MOUNT_DIR}
             if [ $? -ne 0 ]; then
```

Comment:
We should also move this test or remove it if it is no longer needed

Reply:
Hi @mwunderl it's done!

Cheers,

Reply:
Thanks! I'm just going to adjust the whitespace on the if block